client/site-profiler/components/domain-analyzer/styles.scss

```diff
@@ -7,6 +7,7 @@
 .domain-analyzer--form-container {
 	background: #fff;
 	box-shadow: 0 5px 10px 0 #0000000d;
+	border: 1px solid transparent;
```

Comment:
Adding a transparent border to avoid flickering between the error state border and the default state.